projects/opentelemetry-interceptor/src/lib/interceptor/opentelemetry-http.interceptor.ts

```diff
@@ -153,6 +156,32 @@ export class OpenTelemetryHttpInterceptor implements HttpInterceptor {
     );
   }
 
+  /**
+   * Check if {@param url} should be ignored when comparing against {@param ignoredUrls}
+   *
+   * @param ignoredUrls
+   * @param url
+   */
+  private isUrlIgnored(url, ignoredUrls) {
+    if (!ignoredUrls) {
+      return false;
+    }
+    for (const ignoreUrl of ignoredUrls) {
+      if (this.urlMatches(url, ignoreUrl)) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  private urlMatches(url, urlToMatch) {
+    if (typeof urlToMatch === 'string') {
+      return url === urlToMatch;
+    } else {
+      return !!url.match(urlToMatch);
+    }
+  }
+
```

Comment:
You can use `isUrlIgnored` from opentelemetry/core in interceptor and remove this code.

`import { isUrlIgnored } from '@opentelemetry/core';`

Reply:
Done!